xAPI.md

```diff
@@ -3280,10 +3299,10 @@ requested by voidedStatementId.
 Statement when retrieving Statements using explicit or implicit time or sequence based retrieval,
 unless they themselves have been voided, as described in
 [the section on filter conditions for StatementRefs](#queryStatementRef). This includes the
-voiding Statement, which cannot be voided. Reporting tools can identify the presence and
+voiding Statement, which cannot be voided. Clients can identify the presence and
 statementId of any voided Statements by the target of the voiding Statement. 
 
-* Reporting tools wishing to retrieve voided Statements SHOULD request these individually by 
+* Clients wishing to retrieve voided Statements SHOULD request these individually by 
```

Comment:
This isn't part of this change, but this isn't actually a SHOULD. If they want to retrieve them using the xAPI, then there's no other option -- retrieving them and using voidedStatementId are conceptually inseparable. Kinda weird to have a SHOULD (or any requirement at all) on something that literally can't be avoided. Instead of being a requirement, this should just be a comment which says something "To retrieve voided Statements, use voidedStatementId" or even be deleted entirely.